extra-python/pyxtrlock/autobuild/prepare

```diff
@@ -0,0 +1,3 @@
+abinfo "Adding the code of utf-8 into setup.py"
+sed -i '1 i# -*- coding: utf-8 -*' "$SRCDIR"/setup.py
+alias python=python3
```

Comment:
Why is this necessary?

Reply:
Oh, I think the whole `prepare` file is unnecessary. I use this file to solve `SyntaxError: Non-ASCII character` when I didn't know `NOPYTHON2=1`. I will delete `prepare`.